types/burns/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for burns 2.0
+// Project: https://github.com/shalvah/burns#readme
+// Definitions by: Timothy Onyiuke <https://github.com/timolinn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const _Burns: Burns;
+
+export interface ObjectLiteral {
```

Comment:
This isn't needed if you use the built-in `Record` type.

Reply:
Fixed.